manifests/s/staniel359/muffon/2.0.1/staniel359.muffon.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: staniel359.muffon
+PackageVersion: 2.0.1
+Platform:
+- Windows.Desktop
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2023-11-14
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/staniel359/muffon/releases/download/v2.0.1/muffon-2.0.1-win-x64-portable.exe

```

Comment:
Is this really a nullsoft installer, or is it a `portable` binary?

Reply:
The portable version should be moved to another identifier, such as `staniel359.muffon.portable`, which would also resolve the duplicate installer issue.